src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -7,6 +7,7 @@
 using System.Runtime.Versioning;
 using System.Net.Quic;
 using System.IO;
+using System.Linq;
```

Comment:
```suggestion
```

Reply:
oops, I missed this one and merged it, I will remove it when I next touch HTTP3